Events/StarScannedEvent.cs

```diff
@@ -91,7 +93,13 @@ static StarScannedEvent()
 
         public List<Ring> rings { get; private set; }
 
-        public StarScannedEvent(DateTime timestamp, string name, string stellarclass, decimal solarmass, decimal radius, decimal absolutemagnitude, string luminosityclass, long age, decimal temperature, decimal distancefromarrival, decimal? orbitalperiod, decimal rotationperiod, decimal? semimajoraxis, decimal? eccentricity, decimal? orbitalinclination, decimal? periapsis, List<Ring> rings) : base(timestamp, NAME)
+        public decimal? estimatedhabzoneinner { get; private set; }
```

Comment:
CamelCase please when in C# land.

Reply:
I think this one extends out of C# land?

Reply:
hmm, OK.